.github/ISSUE_TEMPLATE/new-ontology-request.md

```diff
@@ -26,6 +26,7 @@ GitHub username:
 
 ## Issue tracker
 
+## Version Controlled Repository
```

Comment:
consider updating wording here

Reply:
Suggestion?

Reply:
Just make PR if you have a suggestion for improvement.